content/applications/general/voip/onsip.rst

```diff
@@ -5,94 +5,180 @@ Use VoIP services in Odoo with OnSIP
 Introduction
 ============
 
-Odoo VoIP can be set up to work together with OnSIP (www.onsip.com). In that case, the installation
-and setup of an Asterisk server is not necessary as the whole infrastructure is hosted and managed
-by OnSIP.
+Odoo VoIP can be set up to work together with `OnSIP <https://www.onsip.com/>`_. In this case, the
+installation and setup of an Asterisk server is not necessary as the whole infrastructure is hosted
```

Comment:
this phrase makes no sense now that we entirely removed the doc about setting up asterisk imho